src/rpc/rawtransaction.cpp

```diff
@@ -1974,11 +2018,28 @@ RPCHelpMan descriptorprocesspsbt()
     bool bip32derivs = request.params[3].isNull() ? true : request.params[3].get_bool();
     bool finalize = request.params[4].isNull() ? true : request.params[4].get_bool();
 
+    std::vector<CTransactionRef> prev_txns;
+    // Parse dependant transactions to populate input UTXOs
+    if (!request.params[5].isNull()) {
+        const UniValue raw_transactions = request.params[5].get_array();
+        prev_txns.reserve(raw_transactions.size());
+
+        for (const auto& rawtx : raw_transactions.getValues()) {
+            CMutableTransaction mtx;
+            if (!DecodeHexTx(mtx, rawtx.get_str())) {
+                throw JSONRPCError(RPC_DESERIALIZATION_ERROR,
+                                   "TX decode failed: " + rawtx.get_str() + " Make sure the prev tx has at least one input.");
+            }
+            prev_txns.emplace_back(MakeTransactionRef(std::move(mtx)));
+        }
+    }
+
```

Comment:
Does it make sense to deduplicate this code piece by accepting `prev_txs, request.params[i]` as inputs to a function?

Reply:
good point, could be used elsewhere too in a follow-up e.g., `testmempoolaccept`, `submitpackage`, etc. done.